imgui-SFML.cpp

```diff
@@ -272,43 +285,54 @@ void Init(sf::Window& window, const sf::Vector2f& displaySize, bool loadDefaultF
                     sf::Cursor::SizeTopLeftBottomRight);
     loadMouseCursor(ImGuiMouseCursor_Hand, sf::Cursor::Hand);
 
-    if (s_fontTexture) {  // delete previously created texture
-        delete s_fontTexture;
+    if (s_currentWindowContext->fontTexture) {  // delete previously created texture
+        delete s_currentWindowContext->fontTexture;
     }
-    s_fontTexture = new sf::Texture;
+
+    s_currentWindowContext->fontTexture = new sf::Texture;
 
     if (loadDefaultFont) {
         // this will load default font automatically
         // No need to call AddDefaultFont
         UpdateFontTexture();
     }
 
-    s_windowHasFocus = window.hasFocus();
+    s_currentWindowContext->windowHasFocus = window.hasFocus();
+}
+
+void SetCurrentWindow(const sf::Window& window) {
+    auto it = std::find_if(begin(windowContexts), end(windowContexts), [&](const WindowContext& ctx) {
```

Comment:
It pains me to write this, but in C++03 we'd probably do this by simple for loop, and I know about stuff like std::bind, but I'd rather not use it.